Towny/src/main/java/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -1598,7 +1598,7 @@ private static void parseJailCommand(CommandSender sender, Town town, String[] s
 			// Set the jail cell if the argument is given.
 			if (split.length == 4 + offset) {
 				cell = MathUtil.getPositiveIntOrThrow(split[3 + offset]);
-				if (!jail.hasJailCell(cell))
+				if (!jail.hasJailCell(Math.max(0, cell - 1)))

```

Comment:
Not sure that is good because Jail#hasJailCell removes 1 from the given input already.